### ISSUE TITLE
How to update simnet using loss_ce_supervised?

### ISSUE THREAD

**Issue by pengzhenjia** [nayeemrizve/OpenLDN#8]
How to update simnet using loss_ce_supervised? It seems that loss_ce_supervised is generated by model_ and has nothing to do with simnet. 
# update simnet parameters
feat, logits = model_(inputs)
logits = de_interleave(logits, 2*args.mu+1)
logits_l = logits[:batch_l[0]]
loss_ce_supervised = F.cross_entropy(logits_l, targets_l)

optimizer_simnet.zero_grad()
loss_ce_supervised.backward()
optimizer_simnet.step()
